Registry/prefab_dependency_viewer.setreg

```diff
@@ -0,0 +1,12 @@
+{
+    "Amazon":
+    {
+        "Gems":
+        {
+            "PrefabDependencyViewer.Editor":
+            {
+                "AutoLoad": false
+            }
+        }
+    }
+}
```

Comment:
If you put this setreg file here in the global O3de registry in the root it will apply it to all projects.  This should cause it to be impossible to load the Prefab Dependency Viewer editor gem module.  Why?

Reply:
This should be in the Registry folder of the Gem.

Reply:
@lawsonamzn To answer your question, the reason "AutoLoad" is false, is because the PrefabDependencyViewer feature isn't complete and this is to prevent the gem from being loaded until it gets complete.

@pjuneja760 I realized @AZMN-koppersr is correct, this can be put in the `Registry` folder of the gem itself.
That directory would be `<engine-root>/Gems/Prefab/Registry`

Reply:
Resolved